### PR TITLE
Longhorn v1.1.3: Raise minimum supported Kubernetes version to v1.18, support Kubernetes v1.22, and and bump CSI sidecar components' version

### PR DESCRIPTION
**Pull request by PhanLe1010**
longhorn/longhorn#2971

### CODE REVIEW

**Comment by PhanLe1010** [comment]
I am doing some more testings

**Comment by PhanLe1010** [comment]
Ready for reviewing

**Review by jenting — APPROVED** [review]
LGTM
Just to make sure we won't encounter v1.2.0 issue again that without specifying the ext4 parameter.
And make sure the corresponding doc be updated as well.

**Comment by PhanLe1010** [comment]
Thanks @jenting 

> Just to make sure we won't encounter v1.2.0 issue again that without specifying the ext4 parameter.

Yes, it is checked

> And make sure the corresponding doc be updated as well.

Doc PR was merged https://github.com/longhorn/website/pull/418

File: deploy/longhorn-images.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-longhornio/csi-attacher:v2.2.1-lh1
-longhornio/csi-node-driver-registrar:v1.2.0-lh1
-longhornio/csi-provisioner:v1.6.0-lh1
-longhornio/csi-resizer:v0.5.1-lh1
-longhornio/csi-snapshotter:v2.1.1-lh1
+longhornio/csi-attacher:v3.2.1
```
Comment (innobead) [inline deploy/longhorn-images.txt:1]: It seems like use the cloned image. Please also make the change to 1.2.x and master.

**Review by innobead — APPROVED** [review]
LGTM

File: deploy/longhorn-images.txt (hunk 1 above)
Comment (innobead) [reply deploy/longhorn-images.txt:1]: ref: https://github.com/longhorn/longhorn/issues/3056 cc @meldafrawi

**Comment by jenting** [comment]
> Thanks @jenting
> 
> > Just to make sure we won't encounter v1.2.0 issue again that without specifying the ext4 parameter.
> 
> Yes, it is checked
> 
> > And make sure the corresponding doc be updated as well.
> 
> Doc PR was merged [longhorn/website#418](https://github.com/longhorn/website/pull/418)

I think you missed to backport this [PR](https://github.com/longhorn/website/pull/358) as well

**Comment by PhanLe1010** [comment]
Thanks @jenting created backport PR https://github.com/longhorn/website/pull/423

**Pull request merged**